client/state/action-types.js

```diff
@@ -506,9 +513,9 @@ export const THEME_UPLOAD_FAILURE = 'THEME_UPLOAD_FAILURE';
 export const THEME_UPLOAD_PROGRESS = 'THEME_UPLOAD_PROGRESS';
 export const THEME_UPLOAD_START = 'THEME_UPLOAD_START';
 export const THEME_UPLOAD_SUCCESS = 'THEME_UPLOAD_SUCCESS';
-export const ACTIVE_THEME_REQUEST = 'ACTIVE_THEME_REQUEST';
-export const ACTIVE_THEME_REQUEST_SUCCESS = 'ACTIVE_THEME_REQUEST_SUCCESS';
-export const ACTIVE_THEME_REQUEST_FAILURE = 'ACTIVE_THEME_REQUEST_FAILURE';
+export const TICKET_CONFIGURATION_REQUEST = 'TICKET_CONFIGURATION_REQUEST';
+export const TICKET_CONFIGURATION_REQUEST_SUCCESS = 'TICKET_CONFIGURATION_REQUEST_SUCCESS';
+export const TICKET_CONFIGURATION_REQUEST_FAILURE = 'TICKET_CONFIGURATION_REQUEST_FAILURE';
```

Comment:
Are these actions still being used? Are they different to the `HELP_TICKET_*` actions?